rbc/tests/test_externals_cmath.py

```diff
@@ -122,11 +130,16 @@ def fn(a, b):
             def fn(a, b, c):
                 return cmath_fn(a, b, c)
 
-        fn.__name__ = fn_name
+        fn.__name__ = fname
```

Comment:
We cannot use the same name of an omnisci SQL function for a UDF. For instance, when `fname == 'cos'`, the corresponding UDF will never be called as omnisci already defines `COS` function.
To demonstrate this, apply

```
diff --git a/rbc/tests/test_externals_cmath.py b/rbc/tests/test_externals_cmath.py
index f8a2707..4bf8e20 100644
--- a/rbc/tests/test_externals_cmath.py
+++ b/rbc/tests/test_externals_cmath.py
@@ -106,7 +106,7 @@ cmath_funcs = (
 
 funcs = {}
 
-
+from rbc.externals.stdio import printf
 def define(jit):
     global funcs
 
@@ -116,10 +116,12 @@ def define(jit):
         retty = str(t[0])
         argtypes = tuple(map(str, t[1]))
         arity = len(argtypes)
-
+        bfname = ('CALLING ' + fname + '\n').encode()
         # define callable
         if arity == 1:
             def fn(a):
+                for c in bfname:
+                    printf("%c", c)
                 return cmath_fn(a)
 
         elif arity == 2:
```
and notice that omnisci server output does not contain `CALLING cos`, for instance.

So, we'll need to restore `fn_name` feature to be sure that rbc test suite will not define UDFs with the same names as existing SQL operators.